test/node/module/authenticate/setClientCertByPath.test.js

```diff
@@ -69,8 +69,8 @@ describe('Connection module', () => {
       const auth = new Auth()
         .setPasswordAuth(paramPasswordAuth)
         .setClientCertByPath(paramClientCertByPath);
-      const conn = new Connection(common.DOMAIN, auth);
-      conn.setProxy(common.PROXY_HOST, common.PROXY_PORT);
+      const conn = new Connection({domain: common.DOMAIN, auth: auth});
+      conn.setProxy({host: common.PROXY_HOST, port: common.PROXY_PORT});
       const appModule = new App(conn);
```

Comment:
App module will be changed to single object params. Update App module first, then update this PR